charts/mentoring/values.yaml

```diff
@@ -0,0 +1,69 @@
+# Default values for mentoring.
+replicaCount: 1
+
+image:
+  repository: 783633885093.dkr.ecr.us-west-2.amazonaws.com/mentoring
+  pullPolicy: IfNotPresent
+  tag: "0.0.1"
+
+fullnameOverride: "mentoring"
+
+serviceAccount:
+  create: true
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: true
+  annotations:
+    kubernetes.io/ingress.class: nginx-mentoring
+  hosts:
+    - host: mentoring.mozilla.com
+      paths:
+        - path: /
+      backend:
+        serviceName: mentoring
+        servicePort: 80
+
+resources:
+  limits:
+    cpu: 500m
+    memory: 1024Mi
+  requests:
+    cpu: 100m
+    memory: 128Mi
+
+autoscaling:
+  enabled: true
+  minReplicas: 1
+  maxReplicas: 3
+  targetCPUUtilizationPercentage: 80
+  targetMemoryUtilizationPercentage: 80
+
+configMap:
+  allowedHosts: '"*"'
+
+externalSecrets: false
```

Comment:
How will we manage these if not using external secrets?

Reply:
For testing the chart in CI in this repo, we won't use external secrets but rather a secret using dummy values.

See [the secrets file](https://github.com/mozilla-it/helm-charts/pull/79/files#diff-efc51e0e267ea8c35139d6ba1a39f6845f32f39e004ea90a532c235d06c251e7) for more information and an example